src/forms/TopComponent/Views/DesktopView.tsx

```diff
@@ -20,13 +20,12 @@ const BIG_SCREEN_BREAKPOINT = BREAKPOINTS.topComponent.bigScreenView + "px";
 
 const DesktopViewColumn = styled(StyledColumn)`
   justify-content: space-between;
-  padding: 24px 20px 0px 20px;
+  padding: 24px 20px 30px 20px;

```

Comment:
This padding 30px makes some weird spaces, specially on the "Form with Layout" example, because it renders the tabs with a lot of space under them. I'd recommend 2 approaches: 1. Return this to 0 and keep the margin-bottom: 30 on the DesktopTitleActionsRow (I've been reading the ticket and it seems the only issue was when no description was added), or 2. Add a conditional to add those 30px when no sections are being propped down.